src/poetry_plugin_export/exporter.py

```diff
@@ -98,17 +95,10 @@ def _export_requirements_txt(self, cwd: Path, output: IO | str) -> None:
         ops = solver.solve().calculate_operations()
         packages = sorted((op.package for op in ops), key=lambda pkg: pkg.name)
 
-        # Get project dependencies.
-        if self._groups is not None:
-            root_package = self._poetry.package.with_dependency_groups(
-                list(self._groups), only=True
-            )
-        else:
-            root_package = self._poetry.package.without_optional_dependency_groups()
-
         for dependency_package in self._poetry.locker.get_project_dependency_packages(
-            project_requires=root_package.all_requires,
-            project_python_marker=root_package.python_marker,
+            project_requires=root.all_requires,
+            project_python_marker=root.python_marker,
+            dev=True,
```

Comment:
```suggestion
```

See: https://github.com/python-poetry/poetry/pull/5413/commits/642775bf9c7ad2ac74e9495fd8290525bb9060c9

Reply:
Committed suggestion to trigger the CI run.